### PR TITLE
String to css unicode

### PR DESCRIPTION
**Pull request by tokenosopher**
Checked on iOS as well, looking good.

### CODE REVIEW

File: src/formatters/stringToCSSUnicode.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const stringToCSSUnicode = (text: string): string =>
```
Comment (ogroppo) [inline src/formatters/stringToCSSUnicode.ts:1]: wait is not exported in index file?

Comment (tokenosopher) [reply src/formatters/stringToCSSUnicode.ts:1]: Ah oops

Comment (tokenosopher) [reply src/formatters/stringToCSSUnicode.ts:1]: Done.

**Review by ogroppo — APPROVED** [review]
LEGGO

**Pull request merged**